Relationship/book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../Relationship/rental'
+
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Kindly create a `rental` in this method, this method should accept `person` and `date` as arguments and inside it should create a `rental` object using `self` and the arguments like this:

```
Rental.new(date, self, person)
```